### PR TITLE
Connect user

### PR DESCRIPTION
**Pull request by ng95junwei**
I left all the changes from before in, but I'm not sure if all of them are relevant.

### CODE REVIEW

**Comment by ng95junwei** [comment]
Fixes #78

File: Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN go build -o main .
 # Final Stage
 FROM alpine
 WORKDIR /app
-COPY --from=golang-builder /build/main .
-COPY --from=react-builder /build/dist ./dist
+COPY --from=golang-builder /build/main /app/main
+COPY --from=react-builder /build/dist/ /app/ui/dist/
```
Comment (subhransu) [inline Dockerfile]: Is this working?

File: data/seed.sql [hunk 2]
```diff
@@ -0,0 +1,93 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 12.1
+-- Dumped by pg_dump version 12.1
+
```
Comment (subhransu) [inline data/seed.sql]: We can move this to a seperate PR.

File: ui/src/shared/components/PostDetail.tsx [hunk 3]
```diff
@@ -65,6 +65,28 @@ const DangerText = styled.span`
...
+  const craftMessage = () => {
+    const messageDialogParameters: MessageDialogParameters = {
+      title: "Contacting",
+      job_poster: props.data.created_by,
+      position: {
+        job_title: props.data.title,
+        placeholder: "Write your message here",
+      },
+      current_user: {
+        // TODO(ivanfoong) populate actual current user
```
Comment (subhransu) [inline ui/src/shared/components/PostDetail.tsx:77]: looks like this is hardcoded. What exactly this `current_user` is doing?

**Review by subhransu — DISMISSED** [review]

File: ui/src/shared/components/PostDetail.tsx (hunk 3 above)
Comment (ng95junwei) [reply ui/src/shared/components/PostDetail.tsx:77]: removed as this is part of #102

File: data/seed.sql (hunk 2 above)
Comment (ng95junwei) [reply data/seed.sql]: removed from PR

File: Dockerfile (hunk 1 above)
Comment (ng95junwei) [reply Dockerfile]: removed from PR

**Review by SheldonCheng — APPROVED** [review]

**Pull request merged**